test_programs/compile_success_empty/ec_baby_jubjub/src/main.nr

```diff
@@ -9,10 +9,11 @@ use std::ec::swcurve::curvegroup::Point as SWG;
 
 use std::ec::montcurve::affine::Point as MGaffine;
 use std::ec::montcurve::curvegroup::Point as MG;
+use std::compat;
 
 fn main() {
     // This test only makes sense if Field is the right prime field.
-    if 21888242871839275222246405745257275088548364400416034343698204186575808495617 == 0 {
+    if compat::is_bn254() {
```

Comment:
I guess this PR is a breaking change... but I guess a good one?

Reply:
Yeah even though we refactoring `is_bn254` to not rely on `p == 0` other programs may rely on this functionality.